api/lib/domain/usecases/create-pro-organizations-with-tags.js

```diff
@@ -59,6 +63,16 @@ module.exports = async function createProOrganizationsWithTags({
     await organizationTagRepository.batchCreate(organizationsTags, domainTransaction);
   });
 
+  createdOrganizations = createdOrganizations.flatMap(({ id, externalId, name }) => {
+    const { organization } = organizationsData.get(externalId);
+    return {
+      email: organization?.email,
+      externalId,
+      id,
+      name,
+    };
+  });
+
   const createdOrganizationsWithEmail = createdOrganizations.filter((organization) => !!organization.email);
```

Comment:
c'est 2 peuvent etre merged ensemble

```suggestion
  const createdOrganizationsWithEmail = createdOrganizations.map(({ id, externalId, name }) => {
    const { organization } = organizationsData.get(externalId);
    return {
      email: organization?.email,
      externalId,
      id,
      name,
    };
  }).filter((organization) => !!organization.email);
```

Reply:
Done ✅